src/platform-implementation-js/dom-driver/gmail/widgets/gmail-mole-view-driver.ts

```diff
@@ -104,7 +104,24 @@ class GmailMoleViewDriver {
 
   show() {
     const doShow = (moleParent: HTMLElement) => {
-      moleParent.insertBefore(this.#element, last(moleParent.children)!);
+      const lastChild = last(moleParent.children)!;
+
+      if (lastChild instanceof HTMLElement) {
+        /**
+         * 2023-10-02 When Google Chat is enabled, we need to keep track of the `order` style property for moles added.
+         * If we don't, we end up with moles added on top of the sidebar because of a chat element in the same parent element as moleParent.
```

Comment:
```suggestion
         * If we don't, we end up with moles added on top of the sidebar because the moleParent is using `order` for layout, and any child without it will be force to the right edge of the page.
```